### ISSUE TITLE
About Training Details

### ISSUE THREAD

**Issue by MinJunKang** [Stability-AI/stable-virtual-camera#66]
Hello, thanks for sharing the code.

I am trying to build training script of SEVA based on previous issues and your comments.

However, I am not sure about several points here.

1. Which lr_scheduler did you use for training? lambda_linear in [here](https://github.com/Stability-AI/generative-models/blob/1659a1c09b0953ad9cc0d480f42e4526c5575b37/sgm/lr_scheduler.py#L113) or constant scheduler that SVD uses?
2. what is the learning rate you used for training? 1e-4?

Thanks!!
